pyCGM_Single/pycgmCalc.py

```diff
@@ -142,12 +232,126 @@ def calcAngles(data,**kargs):
         return r,jcs
 
 def Calc(start,end,data,vsk):
+    """Calculates angles and joint values for marker data in a given range
+    
+    This function is a wrapper around `calcFrames`. It calls `calcFrames`
+    with the given `data` and `vsk` inputs starting at index `start` and 
+    ending at index `end` in `data`.
+
+    Parameters
+    ----------
+    start : int
+        Start index for the range of frames in `data` to calculate
+    end : int
+        End index for the range of frames in `data` to calculate. The data
+        at index `end` is not included.
+    data : array of dict or array
+        List of xyz coordinates of marker positions in a frame. Each 
+        coordinate is a dict where the key is the marker name and the 
+        value is a 3 element array of its xyz coordinate. Can also pass
+        as an array of [labels, data], where labels is a list of
+        marker names and data is list of corresponding xyz coordinates. 
+    vsk : dict or array
+        Dictionary containing subject measurement values, or array of 
+        labels and data [labels, data]. 
+
+    Returns
+    -------
+    angles, jcs : tuple
+        `angles` is an array of the joint angle values. 'jcs' is an array
+        of joint center locations. Indices correspond to frames in the 
+        trial.
+
+    Examples
+    --------
+    First, we load motion capture data from Sample_Static.c3d
+    and subject measurement values from Sample_SM.vsk in 
+    /SampleData/ROM/. 
+
+    >>> from numpy import around
+    >>> from .pycgmIO import loadC3D, loadVSK
+    >>> from .pycgmStatic import getStatic
+    >>> from .pyCGM_Helpers import getfilenames
+    >>> filenames = getfilenames(x=2)
+    >>> c3dFile = filenames[1]
+    >>> vskFile = filenames[2] 
+    >>> result = loadC3D(c3dFile)
+    >>> data = result[0]
+    >>> vskData = loadVSK(vskFile, False)
+    >>> vsk = getStatic(data,vskData,flat_foot=False)
+    
+    A start value of 0 and an end value of 3 indicates that we want
+    to calculate angles for frames 0-2. 
+
+    >>> start = 0
+    >>> end = 3
+    >>> angles, jcs = Calc(start, end, data, vsk)
+
```

Comment:
I don't think this space is needed

Reply:
I removed the space here and other places I had unnecessary spaces.